src/gam.py

```diff
@@ -94,6 +94,12 @@
     {address}
 """
 
+def cp65001(name):
+  if name.lower() == "cp65001":
+    return codecs.lookup("utf-8")
+
+codecs.register(cp65001)
```

Comment:
shouldn't cp650001 be quoted here?

Reply:
No, you are registering a function name.
Here's a on-line version:
codecs.register(lambda name: codecs.lookup('utf-8') if name == 'cp65001' else None)